### PR TITLE
perl: /tmp/access_log was replaced by logging to stdout.

### PR DESCRIPTION
**Pull request by php-coder**
Related PR: https://github.com/openshift/sti-perl/pull/53

@rhcarvalho @mfojtik @mnagy PTAL

Questions:
1) I've copied&pasted the section from PHP page and now I see that all the paragraph is the one line. Hm, may be I should add more newlines to it? It may simplify editing and diff-ing in future.
2) I also see that near my changes there are if/endif conditions. How I can determine if my change is applied to all of the modifications or only to one of them (let's say openshift-origin)? In other words, may be I also should add if/endif.

### CODE REVIEW

**Comment by adellape** [comment]
Hi, @php-coder. I work on the OpenShift docs team. Thanks for submitting!

> 1) I've copied&pasted the section from PHP page and now I see that all the paragraph is the one line. Hm, may be I should add more newlines to it? It may simplify editing and diff-ing in future.

We try to wrap at 80 column lines.

> 2) I also see that near my changes there are if/endif conditions. How I can determine if my change is applied to all of the modifications or only to one of them (let's say openshift-origin)? In other words, may be I also should add if/endif.

Contributing docs on using ifdef/endif are here: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/contributing.adoc#version-management.

The Origin and Enterprise docs share the same source. Usually we would conditionalize for things that are definitely going to stay different between versions (Origin vs Enterprise), for example package/image names. If it's just something going in upstream that will eventually make it into Enterprise (i.e., next release), we (docs team) will do what needs to be done (cherrypick or rebase) to make sure it gets into the Enterprise docs at the right time. For example, I'll be adding the "OSE 3.1" milestone to this PR for our purposes. And I'll merge this docs PR into master after the origin PR has merged usptream.

As for those ifdefs you're seeing In this particular topic, when they added the hot deploy sections recently, it was around the time we were rebasing the Enterprise branch, and I asked them to conditionalize it for now because it was easier at the time. I'll be following up to remove them. Sorry for the confusion.

So to answer your immediate question, you can leave off the ifdefs (like you did in the PHP one), and we'll take care of getting the changes into the Enterprise docs when it's time.

Thanks!

**Comment by adellape** [comment]
Adding [do not merge] til the origin PR is merged.

File: using_images/s2i_images/perl.adoc [hunk 1]
```diff
@@ -109,3 +109,6 @@ command to enter the container:
 After you enter into the running container, your current directory is set to
 `/opt/app-root/src`, where the source code is located.
 endif::openshift-origin[]
+
+== Logs
+Access logs are streamed to standard out and as such they can be viewed via the link:../../cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations[oc logs] command. Error logs are stored in *_/tmp/error_log_* which can be viewed using link:../../dev_guide/executing_remote_commands.html[oc exec] to access the container.
```
Comment (mnagy) [inline using_images/s2i_images/perl.adoc]: s/standard out/standard output/
I think you could also mention `oc rsh` for the container access? Just a suggestion.
Try to wrap to 80 chars as @adellape mentioned.

**Comment by mnagy** [comment]
@php-coder also please rebase and remove the `[do not merge]` prefix

File: using_images/s2i_images/perl.adoc (hunk 1 above)
Comment (php-coder) [inline using_images/s2i_images/perl.adoc]: >s/standard out/standard output/
Done
>I think you could also mention oc rsh for the container access? Just a suggestion.
I believe that is already mentioned in this doc in the of previous sections.
>Try to wrap to 80 chars as @adellape mentioned.
Done

**Comment by php-coder** [comment]
>also please rebase and remove the [do not merge] prefix

I've finished rebase, thanks. I'm not sure about deleting [do not merge] prefix because original PR is still unmerged. Or we should wait for it?

**Comment by mnagy** [comment]
@php-coder oops, sorry, I was looking at a different PR :)

content LGTM

**Comment by mnagy** [comment]
@adellape the PR was merge, this is ready to go.

**Comment by adellape** [comment]
:+1: Thanks. Waiting on Travis.................

**Comment by mfojtik** [comment]
:+1: as well.

**Comment by vikram-redhat** [comment]
Doesn't require a to_followup.

**Pull request merged**